tests/topotests/multicast_pim6_sm_topo1/test_multicast_pim6_sm1.py

```diff
@@ -0,0 +1,1631 @@
+#!/usr/bin/env python
+
+#
+# Copyright (c) 2022 by VMware, Inc. ("VMware")
```

Comment:
Use SPDX-License tag, see other files for an example.

Reply:
Done